testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeJmx.java

```diff
@@ -46,6 +46,7 @@ public void testJmxTablesExposedByDeltaLakeConnectorBackedByGlueMetastore()
     public void testJmxTablesExposedByDeltaLakeConnectorBackedByThriftMetastore()
```

Comment:
nit: would it be possible to add tests with minio maybe?

Reply:
Oh I thought the existing 2 tests testJmxTablesExposedByDeltaLakeConnectorBackedByGlueMetastore and     testJmxTablesExposedByDeltaLakeConnectorBackedByThriftMetastore were sufficient for TestDeltaLakeJmx. If other tests are needed, I can work on separate PR unless necessary.